src/examples/hello-world-backpressure.ts

```diff
@@ -6,11 +6,11 @@
  * License: MIT
  */
 
-import Promise from "ts-promise";
 import Stream from "../lib/index";
+import {delay} from "../lib/util";
 
 Stream.from([1, 2, 3, 4])
-	.map((n) => Promise.resolve(n * 2).delay(1000))
-	.forEach((n) => console.log(n));
+	.map((n: number) => Promise.resolve(n * 2).then((k: number) => delay(1000, k)))
```

Comment:
Please leave out the TS-specific things, as these examples are intended to also be copy-pasteable to e.g. README and should be understandable as 'plain' ES6.

Reply:
ok